### PR TITLE
Update SSH completions to current version

### PR DESCRIPTION
**Pull request by Jakuje**

### CODE REVIEW

**Comment by Jakuje** [comment]
Thank you for the comment. I modified the check to take also older openssh versions into account.

**Comment by Jakuje** [comment]
I am sorry, I wrote the last commit late and probably did not read it correctly. It works like a charm. Thanks for patience :)

File: completions/ssh [hunk 1]
```diff
@@ -108,15 +108,21 @@ _ssh_suboption()
         canonicalizehostname)
             COMPREPLY=( $(compgen -W 'yes no always' -- "$cur") )
             ;;
-        *file|identityagent|include)
+        *file|identityagent|include|controlpath|proxycommand|revokedhostkeys|xauthlocation)
```
Comment (scop) [inline completions/ssh]: Wouldn't proxycommand be better completed with `compgen -c -- "$cur"` than `_filedir`?

Comment (Jakuje) [reply completions/ssh]: You are right. In this context, I noticed that I missed also the `LocalCommand` and `RemoteCommand` options, so I added them to the same branch as `ProxyCommand` in the last commit

**Comment by scop** [comment]
Thanks!

**Pull request merged**